content/en/tracing/guide/ignoring_apm_resources.md

```diff
@@ -2,7 +2,33 @@
 title: Ignoring Unwanted Resources in APM
 ---
 
-A service can handle a variety of requests, some of which you might not want traced or included in trace metrics. An example of this is, possibly, health checks in a web application.
+A service can handle a variety of requests, some of which you might not want traced or included in trace metrics. An example of this is, possibly, health checks in a web application. This documentation covers two main options: Sampling and Filtering.
+
+If you need assistance deciding which option is the most relevant for your use case, contact [Datadog support][1].
+
+# Sampling
+
+If you want the span included in the trace metrics but don't want it ingested, use sampling rules.
+
+## Using sampling rules
+
+The recommended approach is to use sampling rules, which allow you to filter spans based on resource names, service names, tags, and operation names:
+
+```shell
+DD_TRACE_SAMPLING_RULES='[{"resource": "GET healthcheck", "sample_rate": 0.0}]'
+```
+
+Or to filter based on HTTP URL tags:
+
+```shell
+DD_TRACE_SAMPLING_RULES='[{"tags": {"http.url": "http://.*/healthcheck$"}, "sample_rate": 0.0}]'
+```
```

Comment:
In addition to these two quick samples, we should probably link to the `/tracing/trace_ingestion/ingestion_controls` page here. "For more information, see..."

Reply:
I just added a sentence about it at the top of the section :)